extensions/windsurf-raycast/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "windsurf-raycast",
+  "title": "Windsurf Extension",
+  "description": "Quick Raycast actions for Windsurf: open files/folders in Windsurf and manage Windsurf projects.",
+  "icon": "./extension-icon.png",
+  "author": "vikas-bansal",
+  "license": "MIT",
+  "version": "1.0.0",
```

Comment:
Let's remove the `version` field. Raycast does not use this field for version management.